exporters/trace/jaeger/uploader.go

```diff
@@ -41,9 +41,15 @@ type EndpointOption func() (batchUploader, error)
 // For example, localhost:6831.
```

Comment:
Docs need to be updated to describe how the agent endpoint is resolved (i.e. passed value, env var, then defaults).